frontend/lib/ui/localized-outbound-link.tsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import { MessageDescriptor } from "@lingui/core";
+import i18n, { SupportedLocale } from "../i18n";
+import { OutboundLink } from "../analytics/google-analytics";
+import { li18n } from "../i18n-lingui";
+
+export type LocalizedOutboundLinkProps = {
+  /** The human-readable text of the link. */
+  text: MessageDescriptor;
+
+  /** The URLs of the link for each supported locale. */
+  urls: { [k in SupportedLocale]: string };
+};
+
+/**
+ * This component can be used to render an outbound link that
+ * may be different for each supported locale.
+ */
+export const LocalizedOutboundLink: React.FC<LocalizedOutboundLinkProps> = (
```

Comment:
Separating this out into a component might seem like overkill for this one dinky case, but I think it will be useful going forward, as there are numerous places in the Tenant Platform where we link out to external resources that sometimes use different URLs for different locales (met council links come to mind).

We can also extend this component in the future to only take a partial list of localized resources, so that e.g. the component can automatically add text like "(en englais)" to links that are in English but don't have Spanish versions.

Reply:
I like your approach here, especially for readability's sake. I look forward to the clarity of being able to quickly distinguish between which outbound links involve localization and which do not when these components are scattered around a certain page.